kafka-http-connector/main.go

```diff
@@ -182,6 +182,28 @@ func getConfig(metadata kafkaMetadata) (*sarama.Config, error) {
 	return config, nil
 }
 
+func extractControlHeaders(headers []sarama.RecordHeader) ([]byte, bool, []sarama.RecordHeader) {
+	var (
+		key []byte
+	)
+	tombstone := false
+	var cleaned []sarama.RecordHeader
```

Comment:
cleaned is essentially any other key the producer sets that are not keda/fission related, right?

Reply:
Yes - couldn't think of a better name.